server/src/models/Vote.js

```diff
@@ -0,0 +1,43 @@
+const Model = require('./Model.js')
+
+class Vote extends Model{
+    static get tableName(){
```

Comment:
This file looks like it has tab size 4 whereas other files have a tab size of 2. This should be consistent across a codebase and among team members. Sync with your group and check your VSCode settings to ensure they are in line. You may consider using a tool like prettier/ eslint to help with formatting your code for consistency